frontend/packages/metal3-plugin/src/components/host.tsx

```diff
@@ -208,9 +200,34 @@ export const BaremetalHostsPage: React.FC<BaremetalHostsPageProps> = (props) =>
       rowFilters={[hostStatusFilter]}
       createButtonText="Add Host"
       namespace={props.namespace}
-      resources={[hostsResource, machinesResource, nodesResource]}
+      resources={resources}
       flatten={flatten}
       ListComponent={HostList}
+      customData={{
+        hasNodeMaintenanceCapability: props.hasNodeMaintenanceCapability,
+      }}
     />
   );
 };
+
+const hostsPageStateToProps = ({ k8s }) => ({
+  hasNodeMaintenanceCapability: !!k8s.getIn([
```

Comment:
This same code is also in `host-detail` module. Maybe create a function to cut down code duplication?

Reply:
I think it is better to leave it separate. One might want to add something else to the `stateToProps` function and forget that somebody else is also using it.

Reply:
Well, I didn't mean the whole `hostsPageStateToProps` function :smiley: just the `hasNodeMaintenanceCapability` computation from `k8s` object.

Anyway, I'm OK with this as-is.